Sources/CAudioKitEX/Nodes/RenderMeasurerDSP.mm

```diff
@@ -0,0 +1,56 @@
+// Copyright AudioKit. All Rights Reserved. Revision History at http://github.com/AudioKit/AudioKit/
+
+#include "DSPBase.h"
+#import "CAudioKit.h"
+#include <mach/mach_time.h>
+#include "RenderMeasurer.h"
+
+struct RenderMeasurer {
+
+public:
+    RenderMeasurer() {
+        usage = 0;
+    }
+
+    static RenderMeasurer *_Nonnull create() {
+        RenderMeasurer* measurer = new RenderMeasurer();
+        return measurer;
+    }
+
+    _Nonnull AURenderObserver createObserver() {
+        auto sharedThis = std::shared_ptr<RenderMeasurer>(this);
```

Comment:
I believe this pattern is realtime safe, but I would like someone more experienced with C++ to comment on it.